frontend/pages/hosts/details/cards/Software/SelfService/SelfServiceItem/SelfServiceItem.tsx

```diff
@@ -40,7 +40,7 @@ const STATUS_CONFIG: Record<SoftwareInstallStatus, IStatusDisplayConfig> = {
     displayText: "Failed",
     tooltip: ({ lastInstalledAt = "" }) => (
       <>
-        Software failed to install
+        Software failed to install 
         {lastInstalledAt ? `(${dateAgo(lastInstalledAt)})` : ""}. Select{" "}
```

Comment:
I think end of line whitespace may be ignored. This should do the trick though:
```suggestion
        {lastInstalledAt ? ` (${dateAgo(lastInstalledAt)})` : ""}. Select{" "}
```

Reply:
Thank you! Committed your suggestion.